statistical_variables.md

```diff
@@ -6,21 +6,20 @@ nav_order: 4
 
-Many of the Data Commons APIs deal with Data Commons nodes of the type
-[StatisticalVariable](https://datacommons.org/browser/StatisticalVariable). 
+<div markdown="span" class="alert alert-success" role="alert">
+    <i class="fa fa-info-circle"></i> <b>Note:</b>
+    The list of curated statistical variables listed on this page is no longer available.<br>
+    Please use the <a href="https://datacommons.org/tools/statvar#">StatVar Explorer tool</a> which always has the most recent list of statistical variables.
+</div>
 
-A Statistical Variable, StatVar for short, represents different properties on which
-a particular measurement is constrained upon. For instance, if you are looking for
-stats about the [*male population*](https://datacommons/org/browser/Count_Person_Male) in a specific geographic level (a geographic level could be a country, state, county, city, school district), then in data commons there is a 
-StatisticalVariable node that would be defined like:
+Many of the data commons APIs deal with data commons nodes of the type
+[StatisticalVariable](https://datacommons.org/browser/StatisticalVariable). A StatisticalVariable is a node on the data commons knowledge graog that captures any type of metric, statistic or measure that can measured at a place and time. For instance the [number of males in a population](https://datacommons.org/browser/Count_Person_Male).
 
-```
-Node: dcid:Count_Person_Male
-typeOf: dcs:StatisticalVariable
-statType: dcs:measuredValue
-populationType: schema:Person
-gender: dcs:Male
-measuredProperty: dcs:count
-```
+Using the [StatVar Explorer tool](https://datacommons.org/tools/statvar), it is easy for you to get an overview of the different data sources in which the [number of males in a population](https://datacommons.org/browser/Count_Person_Male) is observed as a data point. The tool also enables you to view the available observations made for the StatisticalVariable at different places with example observation time-series to visualize and counts of the number of places for which the statistics are available.
 
-To explore the different Statistical Variables in Data Commons, please visit the [StatVar Explorer tool](https://datacommons.org/tools/statvar)
+The StatVar Explorer tool is also helpful to find exisiting StatisticalVariable definitions which you can re-use in your data import. In the context of the [example StatisticalVariable](https://datacommons.org/browser/Count_Person_Male), you search for it using search terms like "Person Male" and the result will have all the StatisticalVariables that are avaialble on the data commons knowledge graph containing the `populationType = Person` and `gender = Male`. The [example StatisticalVariable](https://datacommons.org/browser/Count_Person_Male) can be accessed by the name of the StatisticalVariable, in this case `Male Population` from the hierarchy of StatisticalVariables (on left-pane of the page) occuring for the `Person With Gender = Male` result. 
```

Comment:
This and the following paragraph is too much information for this page; some of it belongs in the tool. We just want to lead users to the tool from here.

Reply:
Got it! I have made a call to action from the user with a button to redirect them to the tool! 
So, the page will look like this.
![image](https://user-images.githubusercontent.com/5391555/148214409-f13515b4-305d-41f4-a805-11b31e058f2d.png)